### PR TITLE
Bifixer installation with pip

### PR DESCRIPTION
**Pull request by zuny26**
* Added `setup.py`
* Added `scripts/bifixer` that calls the main function from `bifixer.py`
* Moved loomchild segmenter dependency to an [external repo](https://github.com/zuny26/loomchild-segment-py) for easier installation
  * segment submodule removed
  * toolwrapper requirement removed from this repo
  * added `optional-requirements.txt` to include segment as an optional dependency 
  * updated tests

To install bifixer from this fork:
`pip install "bifixer @ git+https://github.com/zuny26/bifixer@setup"`

With loomchild:
`pip install "bifixer[loomchild] @ git+https://github.com/zuny26/bifixer@setup"`

I set the version of bifixer to `0.1` since there is no release on github, but I'm unsure about that, so feel free to change the version to something else :P

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,47 +45,53 @@ If you find Bifixer useful, please consider citing the following paper:
...
 export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/
 ```
 
 ## USAGE ##
 
 ### Bifixer ###
 
-```
+```bash
 usage: bifixer.py [-h] [--scol SCOL] [--tcol TCOL]
```
Comment (ZJaume) [inline README.md:91]: The calls should be updated to just `bifixer` ?

File: README.md [hunk 2]
```diff
@@ -177,30 +184,31 @@ python3.6 bifixer/bifixer.py input-corpus.en-es output-corpus.en-es en es
 
 `bifixer.py` can be parallelized by using your favourite method (for example, GNU parallel)
 
-Suggested usage: 
+Suggested usage:
 
 ```bash
 cat input-corpus.en-es \
     | parallel -j 25 --pipe -k -l 30000 python3.7 bifixer.py -q - - en es \
```
Comment (ZJaume) [inline README.md:191]: Also here

**Review by ZJaume — COMMENTED** [review]
I cannot install with `pip install "bifixer @ https://github.com/zuny26/bifixer@setup"`
```
Collecting bifixer@ https://github.com/zuny26/bifixer@setup
  ERROR: HTTP error 404 while getting https://github.com/zuny26/bifixer@setup
ERROR: Could not install requirement bifixer@ https://github.com/zuny26/bifixer@setup from https://github.com/zuny26/bifixer@setup because of HTTP error 404 Client Error: Not Found for url: https://github.com/zuny26/bifixer@setup for URL https://github.com/zuny26/bifixer@setup
```

**Comment by zuny26** [comment]
> I cannot install with `pip install "bifixer @ https://github.com/zuny26/bifixer@setup"`

Sorry! It should be `pip install "bifixer @ git+https://github.com/zuny26/bifixer@setup"`

**Comment by ZJaume** [comment]
Now it's working! thank you very much Elsa

**Review by ZJaume — APPROVED** [review]

**Review by mbanon — CHANGES_REQUESTED** [review]
Loomchild installation pending because of error

**Review by mbanon — APPROVED** [review]

**Pull request merged**